### DISCUSSION TITLE
阿里云如何添加主机记录"-"

### DISCUSSION THREAD

**Discussion by uncharity** [Jinnrry/PMail#93] [Q&A]
阿里云的DNS解析记录,无法添加主机记录"-",
也无法像我搜到教学中的腾讯云一样添加为空格的主机记录
需要添加的主机记录中也并没有文档演示时的"mail"

**Comment by Jinnrry** [comment]
"-"的意思是这个字段不需要填，不是说要填一个"-"

主机记录是添加a记录的host值，有些地方叫"host"，有些地方叫"主机记录"，有些地方叫"值"，我也不知道阿里云上面叫啥名字
